sema/commons/j2/j2_functions.py

```
@@ -73,16 +76,52 @@
     return xsd_value(str(content), quote, "xsd:double")
 
 
-def xsd_format_date(content, quote, suffix):
+def xsd_format_date(content: any, quote: str, suffix: str) -> str:
     # make rigid date
+    assert not isinstance(content, datetime), "use datetime format for datetime values, or past .date() result"
```

Ruff (0.8.2)

Use of `assert` detected

(S101)